docs/spark-profiling-tool.md

```diff
@@ -86,6 +89,65 @@ There are 3 modes of operation for the Profiling tool:
      com.nvidia.spark.rapids.tool.profiling.ProfileMain  /eventlogDir
     ```
 
+### Auto-Tuner support
```

Comment:
this seems out of place since its really an option.  I think I would rather see this moved down under Profiling tool options and then just mention it in the Collection Mode section above to call it out.